pkg/mountinfo/mountinfo_test.go

```diff
@@ -0,0 +1,597 @@
+			ParentID:       68,
+			StDev:          "0:107",
+			Root:           "/",
+			MountPoint:     "/var/lib/docker/containers/178dc073e1eb58e137d562d28928a187bdc05905ff889b7658dd7a9b3488e494/mounts/shm",
+			MountOptions:   "rw,nosuid,nodev,noexec,relatime",
+			OptionalFields: []string{"shared:159"},
+			FilesystemType: "tmpfs",
+			MountSource:    "shm",
+			SuperOptions:   "rw,size=65536k",
+		},
+		{
+			MountID:        1017,
+			ParentID:       27,
+			StDev:          "0:3",
+			Root:           "net:[4026532775]",
+			MountPoint:     "/run/docker/netns/85c40bc29e30",
+			MountOptions:   "rw",
+			OptionalFields: []string{"shared:168"},
+			FilesystemType: "nsfs",
+			MountSource:    "nsfs",
+			SuperOptions:   "rw",
+		},
+		{
+			MountID:        969,
+			ParentID:       68,
+			StDev:          "0:117",
+			Root:           "/",
+			MountPoint:     "/var/lib/docker/overlay2/e35eafd51f5f1e5a1f055de47bfccae6bb47c57317628507bb00b0981d33d717/merged",
+			MountOptions:   "rw,relatime",
+			OptionalFields: []string{"shared:177"},
+			FilesystemType: "overlay",
+			MountSource:    "overlay",
+			SuperOptions:   "rw,lowerdir=/var/lib/docker/overlay2/l/SRXSWY4NGUH5ZX5EHGZ3R72ZZZ:/var/lib/docker/overlay2/l/4MKSUFUR2WVKNR47LRVGQUHDIX:/var/lib/docker/overlay2/l/CGHCSEQRDDOB7LDRMW6DNKGYKF:/var/lib/docker/overlay2/l/KXCYX6NI6XSXSW3R6YRYOJIXJY:/var/lib/docker/overlay2/l/5SWRNPTBRYPDDHKFPKXBQJ3AGN:/var/lib/docker/overlay2/l/L6OAE4YY2C6BKHH55W42V7T6Y7,upperdir=/var/lib/docker/overlay2/e35eafd51f5f1e5a1f055de47bfccae6bb47c57317628507bb00b0981d33d717/diff,workdir=/var/lib/docker/overlay2/e35eafd51f5f1e5a1f055de47bfccae6bb47c57317628507bb00b0981d33d717/work",
+		},
+		{
+			MountID:        990,
+			ParentID:       68,
+			StDev:          "0:118",
+			Root:           "/",
+			MountPoint:     "/var/lib/docker/containers/eb56406256443e41d885581f36a790155bfa7eab49753a3cbeef6c12226fb7eb/mounts/shm",
+			MountOptions:   "rw,nosuid,nodev,noexec,relatime",
+			OptionalFields: []string{"shared:249"},
+			FilesystemType: "tmpfs",
+			MountSource:    "shm",
+			SuperOptions:   "rw,size=65536k",
+		},
+		{
+			MountID:        1110,
+			ParentID:       27,
+			StDev:          "0:3",
+			Root:           "net:[4026532846]",
+			MountPoint:     "/run/docker/netns/35b7f0885825",
+			MountOptions:   "rw",
+			OptionalFields: []string{"shared:258"},
+			FilesystemType: "nsfs",
+			MountSource:    "nsfs",
+			SuperOptions:   "rw",
+		},
+		{
+			MountID:        657,
+			ParentID:       21,
+			StDev:          "0:98",
+			Root:           "/",
+			MountPoint:     "/sys/fs/bpf",
+			MountOptions:   "rw,relatime",
+			OptionalFields: []string{"shared:267"},
+			FilesystemType: "bpf",
+			MountSource:    "bpffs",
+			SuperOptions:   "rw",
+		},
+	}
+
+	r := bytes.NewBuffer([]byte(mountInfoContent))
+	mountInfos, err := parseMountInfoFile(r)
+	c.Assert(err, IsNil)
+	c.Assert(mountInfos, HasLen, expectedLength)
+	c.Assert(mountInfos, DeepEquals, expectedMountInfos)
```

Comment:
Usually we'll use `comparator.DeepEquals()` from `pkg/comparator`, just because when it fails it provides a nicely-formatted diff between the objects.

Reply:
I guess that you mean `checker.DeepEquals` from `pkg/checker`. Sure, I will do that.